bitnami/redis/templates/podmonitor.yaml

```diff
@@ -48,5 +48,5 @@ spec:
       - {{ include "common.names.namespace" . | quote }}
   selector:
     matchLabels: {{- include "common.labels.matchLabels" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 6 }}
-      app.kubernetes.io/component: metrics
+      app.kubernetes.io/component: node
```

Comment:
Why are you using this label? I think we are not using in latests versions.

Reply:
Good catch! We are using sentinel so I had this one in my pods but I see that it isn't used for the master/replica setup. Removed it to just use the `matchLabels`:

```yaml
        labels:
          app.kubernetes.io/instance: redis2
          app.kubernetes.io/name: redis
```

Should be specific enough